src/api.js

```diff
@@ -110,6 +110,14 @@ var api = {
     "FIREBASE_DEPLOY_URL",
     utils.envOverride("FIREBASE_UPLOAD_URL", "https://deploy.firebase.com")
   ),
+  dynamicLinksOrigin: utils.envOverride(
+    "FIREBASE_DYNAMIC_LINKS_URL",
+    "https://firebasedynamiclinks.googleapis.com"
+  ),
+  dynamicLinksKey: utils.envOverride(
+    "FIREBASE_DYNAMIC_LINKS_KEY",
+    "AIzaSyB6PtY5vuiSB8MNgt20mQffkOlunZnHYiQ"
```

Comment:
Silly question, but I want to confirm - this key is ok to make public, right?

Reply:
Yes, it can only be used to make short links, and then only according to strict URL patterns 😄